app/assets/stylesheets/header.scss

```diff
@@ -145,4 +116,97 @@
       margin-right: 10px;
     }
   }
+
+  @media (max-width: $grid-float-breakpoint-max) {
+
+    .navbar-nav.hidden-xs {
+      margin: 0;
+      margin-top: 6px;
+    }
+
+    .nav-badge {
+      color: $navbar-inverse-link-color;
+      &:hover { color: $navbar-inverse-link-hover-color; }
+      &:hover,
+      &:focus {
+        background-color: transparent;
+      }
+    }
+
+    #navbar-collapse {
+      .form-group,
+      .twitter-typeahead {
+        display: block !important;
+        margin-bottom: 0;
+
+        &,
+        input { width: 100%; }
+      }
+    }
+
+    .nav-badges .dropdown-menu {
+      width: 300px;
+    }
+
+    #user_menu.dropdown-menu {
+      display: block;
```

Comment:
Properties should be ordered background-color, border, box-shadow, display, margin-top, padding, position, width